### PR TITLE
Add embargoed field to Incident

### PR DESCRIPTION
**Pull request by asmorodskyi**
embargoed field allowing to judge if Incident is embargoed or not

### CODE REVIEW

File: migrations/dashboard.sql [hunk 1]
```diff
@@ -13,6 +13,7 @@ CREATE TABLE IF NOT EXISTS incidents (
   approved   BOOLEAN DEFAULT FALSE,
   emu        BOOLEAN DEFAULT FALSE,
   active     BOOLEAN DEFAULT FALSE,
+  crd        BOOLEAN DEFAULT FALSE,
```
Comment (Martchus) [inline migrations/dashboard.sql:16]: I don't think this code will be executed on existing deployments. Supposedly you'll also need an `ALTER TABLE … ADD COLUMN …` line below.

File: lib/Dashboard/Controller/API/Incidents.pm [hunk 2]
```diff
@@ -33,7 +33,8 @@ my $INCIDENT_SPEC = {
     inReviewQAM => {type  => 'boolean'},
     approved    => {type  => 'boolean'},
     emu         => {type  => 'boolean'},
-    isActive    => {type  => 'boolean'}
+    isActive    => {type  => 'boolean'},
+    crd         => {type  => 'string'}
```
Comment (baierjan) [inline lib/Dashboard/Controller/API/Incidents.pm:37]: Is it really a string? The SQL schema says boolean.

File: lib/Dashboard/Model/Incidents.pm [hunk 3]
```diff
@@ -48,7 +48,7 @@ sub build_nr ($self, $inc) {
 
 sub find ($self, $options = {}) {
   my $incidents = $self->pg->db->query(
-    'SELECT number, project, packages, rr_number, review, review_qam, approved, emu, ARRAY_AGG(c.name) as channels
+    'SELECT number, project, packages, rr_number, review, review_qam, approved, emu, crd, ARRAY_AGG(c.name) as channels
```
Comment (kraih) [inline lib/Dashboard/Model/Incidents.pm:51]: And the value needs to be assigned in the code below. Like i said on Slack, it's best to search for an existing field like `emu` and replicate that code.

File: migrations/dashboard.sql (hunk 1 above)
Comment (asmorodskyi) [reply migrations/dashboard.sql:16]: yes will do

File: t/api.t [hunk 4]
```diff
@@ -384,7 +403,8 @@ subtest 'Update individual incidents' => sub {
       inReviewQAM => false,
       approved    => false,
       emu         => false,
-      isActive    => false
+      isActive    => false,
+      embargoed   => false,
```
Comment (kraih) [inline t/api.t:407]: Trailing comma is usually not used.

**Comment by asmorodskyi** [comment]
from my POV it is ready for review

File: t/api.t (hunk 4 above)
Comment (asmorodskyi) [reply t/api.t]: removed

**Review by kraih — APPROVED** [review]
Looks like it should work to me.

**Review by mimi1vx — APPROVED** [review]

**Pull request merged**